src/providers/messages.js

```diff
@@ -10,7 +10,8 @@ const typeMap = {
     [OBJECT_TYPES.OPERATOR_STATUS_TYPE]: DATA_TYPES.DATA_TYPE_TELEMETRY,
     [OBJECT_TYPES.POLL_QUESTION_TYPE]: DATA_TYPES.DATA_TYPE_TELEMETRY,
     [OBJECT_TYPES.ALARMS_TYPE]: DATA_TYPES.DATA_TYPE_ALARMS,
-    [OBJECT_TYPES.GLOBAL_STATUS_TYPE]: DATA_TYPES.DATA_TYPE_GLOBAL_STATUS
+    [OBJECT_TYPES.GLOBAL_STATUS_TYPE]: DATA_TYPES.DATA_TYPE_GLOBAL_STATUS,
+    ['yamcs.mdbchanges']: DATA_TYPES.DATA_TYPE_MDB_CHANGES
```

Comment:
Could we add this to the `OBJECT_TYPES array from "const.js" to be consistent?

Reply:
I added a new type called `MDB_TYPE` instead since adding it to object types didn't really make sense.